.github/workflows/ci.yml

```
@@ -278,6 +282,7 @@ jobs:
     name: Base testsuite (fips)
     needs: build
     runs-on: ubuntu-latest
+    timeout-minutes: 90
```

```suggestion
    timeout-minutes: 60
```

Last run was in 20 min, perhaps 60 is more than sufficient?